src/validation_workflow/api_request.py

```diff
@@ -20,15 +19,8 @@ class ApiTest:
 
     def __init__(self, request_url: str) -> None:
         self.request_url = request_url
-        self.apiHeaders_auth = {"Authorization": "Basic YWRtaW46YWRtaW4="}  # default user/pass "admin/admin" in Base64 format
-        self.apiHeaders_accept = {"Accept": "*/*"}
-        self.apiHeaders_content_type = {"Content-Type": "application/json"}
-        self.apiHeaders = {}
-        self.apiHeaders.update(self.apiHeaders_auth)
-        self.apiHeaders.update(self.apiHeaders_accept)
-        self.apiHeaders.update(self.apiHeaders_content_type)
 
     def api_get(self) -> Any:
-
-        response = requests.get(self.request_url, headers=self.apiHeaders, verify=False)
-        return response.status_code, response.text
+        self.command = ['curl', self.request_url, '-u', 'admin:admin', '--insecure']
+        result = subprocess.run(self.command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        return result.returncode, result.stdout.decode('utf-8')
```

Comment:
Why did we change from requests to curl here?

Reply:
Thanks for the question, @gaiksaya .. it was an interesting finding while working with @peterzhuamazon yesterday. We found that 'sometimes' the HTTP request to detect cluster readiness by using Pyton library 'requests' did not respond correctelly for some reasons. Instead, I turned to use Curl, and it worked all the time.